lib/chef/resource/chocolatey_feature.rb

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright:: 2018, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Chef
+  class Resource
+    class ChocolateyFeature < Chef::Resource
+      resource_name :chocolatey_feature
+
+      description "Use the chocolatey_feature resource to enable and disable Chocolatey features."
+      introduced "x.x.x"
+
+      property :feature_name, String, name_property: true,
+               description: "The name of the Chocolatey feature to enable or disable."
+
+      property :feature_state, [TrueClass, FalseClass], default: false
+
+      load_current_value do
+        current_state = fetch_feature_element(feature_name)
+        current_value_does_not_exist! if current_state.nil?
+
+        feature_name feature_name
+        feature_state current_state == "true"
+      end
+
+      # @param [String] id the feature name
+      # @return [String] the element's value field
+      def fetch_feature_element(name)
+        require "rexml/document" unless defined?(REXML::Document)
```

Comment:
👍 for using rexml over nokogiri

Reply:
@lamont-granquist can't claim that I knew what I was doing here 😄 I unashamedly borrowed the code from the `chocolatey_config` and `chocolatey_source` and just applied what was required to make it work in the context of the feature command.

Reply:
We have rexml because you shamed me @lamont-granquist when I wrote the original resources with nokogiri